pkg/xds/generator/inbound_proxy_generator.go

```diff
@@ -29,6 +30,7 @@ type InboundProxyGenerator struct{}
 
 func (g InboundProxyGenerator) Generate(ctx context.Context, _ *core_xds.ResourceSet, xdsCtx xds_context.Context, proxy *core_xds.Proxy) (*core_xds.ResourceSet, error) {
 	resources := core_xds.NewResourceSet()
+	unifiedNaming := proxy.Metadata.HasFeature(xds_types.FeatureUnifiedResourceNaming)
```

Comment:
nitpick: we usually wrap this in `getNameOrDefault`

Reply:
ack